crates/rooch-types/src/transaction/authenticator.rs

```diff
@@ -627,4 +656,31 @@ mod tests {
         let auth_key2 = AuthenticationKey::multi_ed25519(&multi_pubkey);
         assert_eq!(auth_key, auth_key2);
     }
+
+    #[test]
+    fn test_temp() {
```

Comment:
@jolestar Please help checking this test case. The `serialized` len is 145 instead of 65, the value of `b` is {r:0, s:0, v:0}.

Reply:
The length because serialized is json.

The `assert_eq!(a.signature, b.signature);` failed because the `v` is u8 when serialized, 256 is overflow.

Reply:
<img width="989" alt="Screen Shot 2023-05-23 at 09 11 32" src="https://github.com/rooch-network/rooch/assets/77268/a489ba2b-3925-4598-b383-95d7d4b74494">

I have no idea why they use u64 as the v's type.